manifests/GiorgioTani/PeaZip/7.3.0.yaml

```
@@ -0,0 +1,15 @@
+Id: Giorgiotani.Peazip
+Publisher: Giorgio Tani
+Name: PeaZip
+Version: 7.3.0
+License: LGPLv3
+InstallerType: MSI
+LicenseUrl: https://www.peazip.org/peazip-tos-privacy.html#software_copyright
+AppMoniker: 
+Tags: zip, compression, peazip, giorgio, tani, files, compressed
```

```suggestion
Tags: "zip, compression, peazip, giorgio, tani, files, compressed"
```

use quotes around tags as shown in the manifest spec https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv0.1.md

---

Thanks @superusercode 😄